some refactoring and new stuff for the forgot password

RickBlaas opened the pull request:
added -forgot_password.php and -forgot_password.view.php both unfinished.

 -index.php added the forgot passowrd route.

 - changed all te drowsapp columns to the correct drowssap spelling.

 -login.view.php added the forgot password link.

 -register.php added new user information to the register page. so now we have firstname, lastname and email added.

 -register.view.php added new input fields.

 -script.sql added new columns to the users table. and changed the insert query to no fill the correct values. and also corrected some incorrect syntax

RickBlaas committed:
added -forgot_password.php and -forgot_password.view.php
 both unfinished.

 -index.php added the forgot passowrd route.

 - changed all te drowsapp columns to the correct drowssap spelling.

 -login.view.php added the forgot password link.

 -register.php added new user information to the register page. so now we have firstname, lastname and email added.

 -register.view.php added new input fields.

 -script.sql added new columns to the users table. and changed the insert query to no fill the correct values. and also corrected some incorrect syntax

Pluers commented on env.ini:
wat doet de mail eigelijk?

RickBlaas replied:
een mail versturen. maar het werkt nog niet. ga ik morgen oplossen

Pluers replied:
alright

Pluers committed:
Update script.sql

removed first part because that already happens if the user doesn't have the script yet
